.istanbul.yml

```diff
@@ -0,0 +1,11 @@
+verbose: false
```

Comment:
gents, I don't _understand_ the advantage of this file ...
Which reminds me, do either of you have time to add as section on `.istanbul.yml` file and what it's useful for? see: https://github.com/dwyl/learn-istanbul/issues/9 (_thanks!!_)